src/ansible_compat/constants.py

```diff
@@ -33,8 +33,8 @@
 role_name: my_name  # if absent directory name hosting role is used instead
 namespace: my_galaxy_namespace  # if absent, author is used instead
 
-Namespace: https://galaxy.ansible.com/docs/contributing/namespaces.html#galaxy-namespace-limitations
-Role: https://galaxy.ansible.com/docs/contributing/creating_role.html#role-names
+Namespace: https://ansible.readthedocs.io/projects/galaxy-ng/en/latest/usage_guide/collections/#permissions
```

Comment:
@tanwigeetika1618 please update the link to point to `[namespace](https://ansible.readthedocs.io/projects/galaxy-ng/en/latest/usage_guide/collections/#namespaces)` instead of `permissions`